src/common/typedefs/navigator.d.ts

```diff
@@ -0,0 +1,11 @@
+declare global {
+  interface WindowControlsOverlay {
+    visible: boolean
+  }
+
+  interface Navigator {
+    windowControlsOverlay: WindowControlsOverlay
```

Comment:
This typedef allows us to later access this property via the standard dot notation
We can remove it once this type is added to the official types package (TS will actually error once this is the case)